database/generate-data.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs');
+const data = require('./data.js');
+// generate data
+// write to csv file
+
+const generateData = (field, fileName) => {
+  // i: string, and string
+  // o: side effect, writes field content to fileName
+
+  let info = field.split('\n').join(',');
+  fs.writeFile(`${fileName}.txt`, info, (err) => {
+    if (err) {
+      console.log('there was an error!');
+    } else {
+      console.log('write was successful!');
+    }
+  })
+
+};
+
+const getRandomIndex = (num) => {
+  // i: number
+  // o: number, random between 0 and num
+
+  function getRandomIntInclusive(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min; //The maximum is inclusive and the minimum is inclusive 
+  };
+
+  return getRandomIntInclusive(0, num);
+}
+
+const capitalizeFirstLetter = (word) => {
+  // i: string, word
+  // o: string, word with first letter capitalized
+
+  let first = word[0];
+  let remainder = word.slice(1);
+
+  return first.toUpperCase() + remainder;
+}
+
+const generateExperience = () => {
+  // i: nothing
+  // o: string
+
+  let result = ``;
+
+  for (let i = 0; i <= 100; i++) {
```

Comment:
I think the Airbnb style guide doesn't recommend using unary operators, they suggest using (i += 1) instead